cli/defenseclaw/tui/panels/setup.py

```
@@ -2108,6 +2642,7 @@ def wizard_form_defs(
     SetupWizard.GUARDRAIL: lambda overrides, cfg: _guardrail_wizard_fields_for(overrides, cfg),
     SetupWizard.GUARDRAIL_ACTIONS: lambda overrides, cfg: _guardrail_actions_wizard_fields(overrides, cfg),
     SetupWizard.CUSTOM_PROVIDERS: lambda overrides, cfg: _custom_providers_fields_for(overrides),
+    SetupWizard.REDACTION: lambda overrides, cfg: _redaction_wizard_fields_for(overrides),
```

Ruff (0.16.1)

[warning] Unused lambda argument: `cfg`

(ARG005)